### PR TITLE
feature/git

### PR DESCRIPTION
**Pull request by joelsrzalvarez**
add git commands to my folder

### CODE REVIEW

File: staff/joel-suarez/git/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+### Moving for branches
+Use this command if you want to switch up to another branch
+```sh
+$ git checkout "branch"
+```
+
+### Delete branch
+Use this command if you want to delete a branch
+```sh
+$ git delete -D "branch"
```
Comment (Barrufat) [inline staff/joel-suarez/git/README.md]: Check this command, it's not correct 🤯

File: staff/joel-suarez/git/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# GIT Coomands
+
+These are some of my git commands
+
+### How to init a git repository
+```sh
+$ git init
+$ git remote add origin 'github url'>
+$ git add 'folder to add'>
+$ git commit -m "message"
```
Comment (Barrufat) [inline staff/joel-suarez/git/README.md:10]: Add specific sections to define and explain the "add" & "commit" commands 👻

File: staff/joel-suarez/git/README.md (hunk 1 above)
Comment (joelsrzalvarez) [reply staff/joel-suarez/git/README.md]: Done, thanks you for the correction!

File: staff/joel-suarez/git/README.md (hunk 2 above)
Comment (joelsrzalvarez) [reply staff/joel-suarez/git/README.md:10]: Done I added comments to the add & commit commands.

Comment (Barrufat) [reply staff/joel-suarez/git/README.md:10]: 🚀🚀😁

**Pull request merged**